dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ﻿# CHANGELOG
+# 1.11.3 - 2024-12-06
```

Comment:
Please can you remove the 1.11.3 changelog and revert the csproj back to 1.11.2.

This work will be part of the larger 1.12.0 release (#184913).

Reply:
reverted 👍